### DISCUSSION TITLE
Pixart-delta with LCM-LoRA

### DISCUSSION THREAD

**Discussion by youngwanLEE** [PixArt-alpha/PixArt-alpha#77] [Q&A]
Hi, 

I'm impressed by your amazing work :)

Have you tried to train pixart-lcm-lora with your [training script](https://github.com/PixArt-alpha/PixArt-alpha/blob/master/train_scripts/train_pixart_lora_hf.py)? If so, I wonder the generation quality compared to the already released [Pixart-LCM](https://huggingface.co/spaces/PixArt-alpha/PixArt-LCM) model.

Thanks in advance.

**Comment by lawrence-cj** [comment]
Actually, [here](https://github.com/PixArt-alpha/PixArt-alpha/blob/master/train_scripts/train_pixart_lcm_lora.py) is the pixart-LCM-LoRA training script. We tried it once, it works but the quality is not as good as fully fine-tuned. I don't know if the Pixart-lora training has any problem. Looking forward to hearing your insight.

**Reply by Ghy1209** [reply]
Hi. can you share the data structure for the pixart-LCM-LoRA?
